src/storePrefetchUtils.js

```diff
@@ -0,0 +1,132 @@
+import { matchPath } from 'react-router';
+
+import config from '@plone/volto/registry';
+
+const PREFETCH_ROUTER_LOCATION_CHANGE = 'PREFETCH_ROUTER_LOCATION_CHANGE';
+
+const matchCurrentPath = (path) => {
+  // Delayed import is necessary to allow accessing the config from routes.
+  // If this is done in import time it would happen before the config is customized.
+  const routes = require('~/routes').default;
+  const defaultRoutes = routes[0].routes;
+
+  // const pathsList = ['/', '/**'];
+  let alreadyMatched;
+
+  for (let pathOption of defaultRoutes) {
+    const match = matchPath(path, pathOption);
+    // console.debug('pathOption', alreadyMatched, path, pathOption, match);
+    if (
+      match &&
+      !alreadyMatched &&
+      (match.path === '/**' || match.path === '/')
```

Comment:
This is not needed, [nonContentRoutes](https://github.com/plone/volto/blob/555ebab6d325053eac2731f9b0bb8389448973b0/src/config/index.js#L99)exists.